compiler/luci/import/include/luci/Import/GraphBuilderRegistry.h

```diff
@@ -61,6 +67,17 @@ class GraphBuilderRegistry final : public GraphBuilderSource
     return _builder_map.at(op).get();
   }
 
+  /**
+   * @brief Returns registered NodeBuilderBase pointer for type or nullptr if not registered
+   */
+  const NodeBuilderBase *lookup(NodeBuilderType type) const final
+  {
+    if (_tensor_builders.at(uint32_t(type)) == nullptr)
```

Comment:
I don't think `at(uint32_t(type))` is safe. maybe use `map` ?

Reply:
`std::map` is applied instead `std::array`, `at(uint32_t(type))` usage removed.